.github/workflows/docs.yml

```
@@ -26,91 +26,38 @@ on:
       - ".github/workflows/docs.yml"
 
 permissions:
-  contents: write
+  contents: read
 
 concurrency:
   group: ${{ github.workflow }}-${{ github.repository }}-${{ github.head_ref || github.sha }}
-  cancel-in-progress: true
+  cancel-in-progress: ${{github.ref != 'refs/heads/presto-0.293-clp-connector'}}
 
 jobs:
   build_docs:
-    name: Build and Push
-    runs-on: 8-core-ubuntu
-    container: ghcr.io/facebookincubator/velox-dev:centos9
+    name: Build
+    runs-on: ubuntu-latest
     env:
       CCACHE_DIR: "/tmp/ccache"
     steps:
-
-      - name: "Restore ccache"
-        if: false
-        uses: apache/infrastructure-actions/stash/restore@3354c1565d4b0e335b78a76aedd82153a9e144d4
-        id: restore-cache
-        with:
-          path: "${{ env.CCACHE_DIR }}"
-          key: ccache-wheels-8-core-ubuntu
-
-      - name: Checkout
-        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
         with:
           fetch-depth: 0
           persist-credentials: true
```

```suggestion
          persist-credentials: true

```

Just to revert unnecessary changes.